tools/java/src/org/apache/bazel/cppcheck/CppCheck.java

```diff
@@ -113,7 +114,7 @@ private static void runChecker(List<String> command) throws IOException {
 
     if (cppcheck.exitValue() == 1) {
       LOG.warning("cppcheck detected bad cpp files.");
-      System.exit(1);
+      throw new RuntimeException("cppcheck detected bad cpp files.");
```

Comment:
Just curious.  What made you add this in for the PR?

Reply:
This is to fix a code styling issue that prevents the Heron docker image from being built.